.github/workflows/macos.yml

```diff
@@ -205,24 +199,34 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
       with:
-        python-version: ${{ matrix.python-version }}
-        cache: 'pip'
+        python-version: "3.13"
 
-    - name: Build and Install SciPy
+    - name: Make gfortran-13 on runner image usable
       run: |
-        sudo xcode-select -s /Applications/Xcode_16.app
+        # Ensure we use gfortran-13 and that its runtime is on the library search path
+        FC=$(which gfortran-13)
+        GFORTRAN_LIB=$(dirname `$FC --print-file-name libgfortran.dylib`)
+        echo DYLD_LIBRARY_PATH=$GFORTRAN_LIB >> "$GITHUB_ENV"
+        echo FC=$FC >> "$GITHUB_ENV"
 
-        git submodule update --init
-        GFORTRAN_LOC=$(which gfortran-13)
-        ln -s $GFORTRAN_LOC gfortran
-        export PATH=$PWD:$PATH
+    - name: Install dependencies from PyPI
+      run: |
+        # These are the required build and test dependencies only
+        pip install meson cython pythran pybind11 ninja numpy spin pooch pytest hypothesis
 
-        # Ensure we have gfortran dylib
-        GFORTRAN_LIB=$(dirname `gfortran --print-file-name libgfortran.dylib`)
-        export DYLD_LIBRARY_PATH=$GFORTRAN_LIB
+    - name: Build SciPy with Accelerate (LP64)
+      if: matrix.blas-interface == 'LP64'
+      run: |
+        # The b_ndebug flag is a workaround for `_LIBCPP_ENABLE_ASSERTIONS` warnings, can
+        # be removed once Meson 1.8.3 is out with the fix (see meson#14440).
+        spin build --with-accelerate -S-Db_ndebug=true
```

Comment:
unrelated - is it worth making the `--with-accelerate` default for spin building? Instead having a `--with-openblas`  flag or similar?